packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -63,7 +85,7 @@ export const getAvailableContractTypes = (contract_types_list, unsupported_list)
  * @param {object} contract_types_list  - list of all contracts
  * @param {array}  filtered_items_array - list of filtered contract category names and/or contract types names
  */
-export const getFilteredList = (contract_types_list, filtered_items_array) => {
+/*export const getFilteredList = (contract_types_list, filtered_items_array: Array<string>) => {
```

Comment:
I commented `getFilteredList`, `flatten` and `getContractsList` functions as they are not being used anywhere. I wasn't able to fined files where they were imported. Not sure if I should delete it as we might need them in future.